notus/scanner/models/os/debian.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2021-2022 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+import re
+from typing import Optional
+from paramiko import SSHClient
+
+from .distribution import Distribution
+
+logger = logging.getLogger(__name__)
+
+
+_debian_re = re.compile(r"^([0-9]+)([0-9.]+)")
+
+
+class Debian(Distribution):
+    @staticmethod
+    def get_os(ssh: SSHClient) -> Optional[str]:
+        # Try Debian
+        rls = Debian.command(ssh, "cat /etc/debian_version")
+        if (
+            _debian_re.match(rls)
+            or rls.find("buster/sid") >= 0
+            or rls.find("bullseye/sid") >= 0
+        ):
+            versions = _debian_re.match(rls).groups()
```

Comment:
match is defined as optional so please verify if it returns something before calling .groups (same on other os implementation)